deluge/core/torrentmanager.py

```diff
@@ -1349,5 +1351,10 @@ def torrents_status_update(self, torrent_ids, keys, diff=False):
         else:
             # Ask libtorrent for status update
             self.torrents_status_requests.insert(0, (d, torrent_ids, keys, diff))
-            self.session.post_torrent_updates()
+            if lt_1_1_compat:
+                # libtorrent 1.1 takes a combination of status_flags_t`s here
+                # like torrent_handle::status()
+                self.session.post_torrent_updates(0xffffffff)
```

Comment:
You need to pass one parameter though (although it has a default value in C++). lt bug?